src/main/java/io/scim2/swagger/client/ApiClient.java

```diff
@@ -0,0 +1,1249 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.scim2.swagger.client;
+
+import com.squareup.okhttp.*;
+import com.squareup.okhttp.internal.http.HttpMethod;
+import com.squareup.okhttp.logging.HttpLoggingInterceptor;
+import com.squareup.okhttp.logging.HttpLoggingInterceptor.Level;
+import io.scim2.swagger.client.auth.Authentication;
+import io.scim2.swagger.client.auth.ApiKeyAuth;
+import io.scim2.swagger.client.auth.HttpBasicAuth;
+import io.scim2.swagger.client.auth.OAuth;
+import okio.BufferedSink;
+import okio.Okio;
+
+import javax.net.ssl.*;
```

Comment:
Use specific import packages instead of import all the packages. Please check this throughout the PR